src/main/java/com/petfinder/controller/UserController.java

```diff
@@ -0,0 +1,56 @@
+package com.petfinder.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import com.petfinder.exception.EmailExistsException;
+import com.petfinder.exception.InvalidEmailException;
+import com.petfinder.exception.LoginExistsException;
+import com.petfinder.exception.PasswordsDoesNotMatchException;
+import com.petfinder.service.UserService;
+
+@Controller
+public class UserController {
+
+	private String status = "";
+	
+	@Autowired
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/matikz1110/PetFinder/file/2526890573/issues/source?bid=3239464&fileBranchId=3282301#l21)